build/brigade.js

```diff
@@ -124,7 +124,7 @@ function testIntegration(e, p) {
     "cd /src",
     "trap 'make -f Makefile.kind delete-kind-cluster' EXIT",
     `make -f Makefile.kind create-kind-cluster`,
-    "make start-local-docker-registry test-integration stop-local-docker-registry"
+    "make test-integration"
```

Comment:
I think we have come to the point where we have firmly settled on azure pipelines (especially now that we don't have a cluster anymore to run on) and it may be time to retire the brigade script. What do you think?

Reply:
Yea 😢 you are right.  I'll issue a follow-up PR to remove these files.  I realize I can maintain local copies for my Brigade project for Porter to test with.